Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -462,6 +468,15 @@ public void CreateRig()
                 return;
             }
 
+            // Record what the initial size of the bounds override
+            // was when we constructed the rig, so we can restore
+            // it after we destructively edit the size with the
+            // BoxPadding (#7997)
```

Comment:
Do you know if issue numbers persist across forks (i.e. if this gets forked/moved does this still remain relevant - I assume not) - if not it might be better to have a full link to the issue (to preserve the project/org/issuenumber context)

Reply:
Ok, fixed.